BarcodeScanning.Native.Maui/BarcodeScanning.Native.Maui.csproj

```diff
@@ -25,6 +26,10 @@
 	<ItemGroup>
 		<PackageReference Include="Microsoft.Maui.Controls" Version="$(MauiVersion)" />
 		<PackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="$(MauiVersion)" />
+		<PackageReference Include="DependencyPropertyGenerator" Version="1.5.0-beta.1">
```

Comment:
I don't think this project should consume beta (pre-release) versions of nuget packages. Maybe wait, until it's released?